.github/workflows/mend-slack.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Mend Slack Notifier'
+
+on:
+  discussion:
+    types: [labeled]
+
+permissions:
+  discussions: read
```

Comment:
This is probably not even needed, but doesn’t really hurt either.

Reply:
this ensures that we only have this limited permission